services/src/main/java/org/keycloak/services/resources/admin/IdentityProvidersResource.java

```diff
@@ -92,8 +95,8 @@ public IdentityProvidersResource(RealmModel realm, KeycloakSession session, Admi
     @NoCache
     @Produces(MediaType.APPLICATION_JSON)
     @Tag(name = KeycloakOpenAPI.Admin.Tags.IDENTITY_PROVIDERS)
-    @Operation( summary = "Get identity providers")
-    public Response getIdentityProviders(@Parameter(description = "Provider id") @PathParam("provider_id") String providerId) {
+    @Operation( summary = "Get the identity provider factory for that provider id")
+    public Response getIdentityProviders(@Parameter(description = "The provider id to get the factory") @PathParam("provider_id") String providerId) {
```

Comment:
Perhaps this can also be less generic then 'Response'? Is there no way we can do a `IdentityProviderRepresentation` as a response?